CommunityFeatures.md

```diff
@@ -36,9 +36,8 @@ Synchronization modes accessible through the "LFO SYNC" shortcut.
  - ([#103]) adds a new filter in the low-pass slot, a state-variable filter. This filter has significantly less distortion than the ladder filters, think sequential vs. moog. Cutoff and resonance ranges are subject to change with further testing.
 
 ## New behaviors
-
-### Instrument Keyboard View
- - ([#46]) Note offset between rows is now configurable by holding shift and using the horizontal encoder. This allows e.g. an isomorphic keyboard layout by setting the row offset to 12. The setting is saved per clip in the song file.
+### Song View Row Colors
```

Comment:
(minor nitpicks sorry, but I figure you're going to be in the area anyway...)

Should remove "Row Colors" here, since this section will have all the changes for "Song View"